propolis/Cargo.toml

```diff
@@ -21,6 +21,11 @@ viona_api = { path = "../viona-api" }
 usdt = "0.1.11"
 tokio = { version = "1", features = ["full"] }
 futures = "0.3"
+crucible = { git = "https://github.com/oxidecomputer/crucible", rev = "adb2662b", optional = true }
```

Comment:
Would it make sense to have a propolis branch or similar on the crucible side?

Reply:
I'd rather it go off a rev of main